### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by nikever**
Refactoring the REAMDE of the module and packages to be consistent with the new documentation format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,361 @@
...
+If you are new to KFD please refer to the [official documentation][kfd-docs] on how to get started with KFD.
 
-- [aws-velero](modules/aws-velero): Creates AWS resources and Kubernetes CRDs needed to persist backups.
-- [eks-velero](modules/eks-velero): Creates AWS resources and Kubernetes CRDs needed to persist backups
-from an eks cluster.
-- [azure-velero](modules/azure-velero): Creates Azure resources and Kubernetes CRDs needed to persist backups.
-- [gcp-velero](modules/gcp-velero): Creates GCP resources and Kubernetes CRDs needed to persist backups.
+## Overview
 
+**Kubernetes Fury DR** module is based on [Velero][velero-page] and Velero Restic.
```
Comment (ralgozino) [inline README.md:18]: link to velero restic maybe?

File: README.md [hunk 2]
```diff
@@ -1,55 +1,361 @@
...
+|                      Package                      | Version |                                   Description                                    |
+| ------------------------------------------------- | ------- | -------------------------------------------------------------------------------- |
+| [velero](katalog/velero)                          | `1.7.1` | Backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. |
+
+The velero package contains the following additional components:
+
+|                     Component                     |                      Description                      |
+| ------------------------------------------------- | ----------------------------------------------------- |
+| [velero-restic](katalog/velero/velero-restic)     | Incremental backup and restore of Kubernetes volumes. |
+| [velero-schedule](katalog/velero/velero-schedule) | Common schedules for backup                           |
```
Comment (ralgozino) [inline README.md:47]: The package/folder name is actually `velero-schedules`, with a trailing `s` (also for the link)

File: README.md [hunk 3]
```diff
@@ -1,55 +1,361 @@
...
+
+### Integration with cloud providers
+
+Use the following Velero Plugins to integrate Velero with cloud providers:
+
+|                   Plugin                    |                      Description                      |
+| ------------------------------------------- | ----------------------------------------------------- |
+| [velero-aws](katalog/velero/velero-aws)     | Plugins to support running Velero on AWS  |
+| [velero-gcp](katalog/velero/velero-gcp)     | Plugins to support running Velero on GCP |
+| [velero-azure](katalog/velero/velero-azure) | Plugins to support running Velero on Azure |
```
Comment (ralgozino) [inline README.md:57]: missing nice formatting in the las column :-P

File: README.md [hunk 4]
```diff
@@ -1,55 +1,361 @@
...
-- :warning: : module version: `v1.9.0` and Kubernetes Version: `1.23.x`. It works as expected. Marked as warning
-because it is not officially supported by [SIGHUP](https://sighup.io).
+| Kubernetes Version |   Compatibility    |                        Notes                        |
+| ------------------ | :----------------: | --------------------------------------------------- |
+| `1.20.x`           | :white_check_mark: | No known issues                                     |
+| `1.21.x`           | :white_check_mark: | No known issues                                     |
+| `1.22.x`           | :white_check_mark: | No known issues                                     |
+| `1.23.x`           |     :warning:      | Conformance tests passed. Not officially supported. |
+
+Check the [compatibility matrix][compatibility-matrix] for additional informations about previous releases of the modules.
```
Comment (ralgozino) [inline README.md:77]: informations -> information (is an uncountable)

File: README.md [hunk 5]
```diff
@@ -1,55 +1,361 @@
...
+## Usage
+
+**Kubernetes Fury DR** deployment depends on the environment.
+
+|               Environment               |   Storage Backend    |                  Velero Plugin                  |           Terraform Module           |
+| --------------------------------------- | -------------------- | ----------------------------------------------- | ------------------------------------ |
+| [Velero on AWS](#velero-on-aws)         | S3 Bucket            | [velero-aws](katalog/velero/velero-aws)         | [aws-velero](modules/aws-velero)     |
+| [Velero on GCP](#velero-on-gcp)         | GCS                  | [velero-gcp](katalog/velero/velero-gcp)         | [gcp-velero](modules/gcp-velero)     |
+| [Velero on Azure](#velero-on-azure)     | AZ Storage Container | [velero-azure](katalog/velero/velero-azure)     | [azure-velero](modules/azure-velero) |
+| [Velero on-premise](#velero-on-premise) | MinIo                | [velero-on-prem](katalog/velero/velero-on-prem) | `/`                                  |
```
Comment (ralgozino) [inline README.md:88]: `on-premises`, note the trailing `s`

File: README.md [hunk 6]
```diff
@@ -1,55 +1,361 @@
...
+| [furyctl][furyctl-repo]     | `>=0.6.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo].     |
+| [kustomize][kustomize-repo] | `>=3.5.0` | Packages are customized using `kustomize`. To learn how to create your customization layer with `kustomize`, please refer to the [repository][kustomize-repo]. |
+| [terraform][terraform-page] | `=0.15.4` | Additional infrastructure is deployed using `terraform`.                                                                                                       |
+
+### Velero on AWS
+
+Velero on AWS is based on the [AWS Velero Plugin][velero-aws-plugin-repo].
+
+It requires the secret `cloud-credentials` in the `kube-system` namespace containing a service account with appropriate credentials.
+In alternative, the module supports [authentication via IAM Roles][aws-docs-iam-roles].
```
Comment (ralgozino) [inline README.md:103]: In alternative -> As an alternative?

File: README.md [hunk 7]
```diff
@@ -1,55 +1,361 @@
...
+modules:
+  - name: dr/aws-velero
+    version: "v1.9.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+2. Execute `furyctl vendor -H` to download the packages
+
+3. Inspect the download packages under `./vendor/katalog/velero`.
```
Comment (ralgozino) [inline README.md:127]: download->downloaded

File: README.md [hunk 8]
```diff
@@ -1,55 +1,361 @@
...
+
+```hcl
+module "velero" {
+  source             = "path/to/vendor/modules/aws-velero"
+  backup_bucket_name = "my-cluster-velero"
+  project            = "sighup-staging"
+}
+```
+
+> More information of modules inputs can be found in the [aws-velero](modules/aws-velero) module documentation
```
Comment (ralgozino) [inline README.md:139]: More information of modules inputs -> More information on modules inputs

File: README.md [hunk 9]
```diff
@@ -1,55 +1,361 @@
...
+```bash
+kustomize build . | kubectl apply -f -
+```
+
+### Velero on GCP
+
+Velero on GCP is based on the [Velero GCP Plugin][velero-gcp-plugin-repo].
+
+It requires the secret `cloud-credentials` in the `kube-system` namespace containing a service account with appropriate credentials.
+In alternative, the module supports workload identity.
```
Comment (ralgozino) [inline README.md:163]: As an alternative?

File: README.md [hunk 10]
```diff
@@ -1,55 +1,361 @@
...
+modules:
+  - name: dr/gcp-velero
+    version: "v1.9.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+2. Execute `furyctl vendor -H` to download the packages
+
+3. Inspect the download packages under `./vendor/katalog/velero`.
```
Comment (ralgozino) [inline README.md:189]: download -> downloaded

File: README.md [hunk 11]
```diff
@@ -1,55 +1,361 @@
...
+
+```hcl
+module "velero" {
+  source             = "path/to/vendor/modules/azure-velero"
+  backup_bucket_name = "my-cluster-velero"
+  project            = "sighup-staging"
+}
+```
+
+> More information of modules inputs can be found in the [azure-velero](modules/azure-velero) module documentation
```
Comment (ralgozino) [inline README.md:260]: More information of modules inputs -> More information on modules inputs

File: README.md [hunk 12]
```diff
@@ -1,55 +1,361 @@
...
+- ./vendor/katalog/velero/velero-schedules
+```
+
+6. To deploy the packages to your cluster, execute:
+
+```bash
+kustomize build . | kubectl apply -f -
+```
+
+### Velero on-premise
```
Comment (ralgozino) [inline README.md:279]: on-premises

File: README.md [hunk 13]
```diff
@@ -1,55 +1,361 @@
...
+
+[velero-on-prem][kfd-velero-on-prem] deploys a [MinIO][minio-page] in-cluster instance as an object storage backend for Velero.
+
+Please note that the MinIO server is running in the same cluster that is being backed up.
+
+To deploy velero on-premise:
```
Comment (ralgozino) [inline README.md:285]: on-premises

File: README.md [hunk 14]
```diff
@@ -1,55 +1,361 @@
...
+    version: "v1.9.0"
+  - name: velero/velero-schedules
+    version: "v1.9.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+2. Execute `furyctl vendor -H` to download the packages
+
+3. Inspect the download packages under `./vendor/katalog/velero`.
```
Comment (ralgozino) [inline README.md:303]: download -> downloaded

File: katalog/velero/README.md [hunk 15]
```diff
@@ -171,4 +165,6 @@ bases:
 ```
 
 More information about
-[velero schedules](https://github.com/vmware-tanzu/velero/blob/master/site/docs/master/api-types/schedule.md)
\ No newline at end of file
+[velero schedules](https://github.com/vmware-tanzu/velero/blob/master/site/docs/master/api-types/schedule.md)
```
Comment (ralgozino) [inline katalog/velero/README.md]: join this 2 lines maybe

**Review by ralgozino — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nikever) [reply README.md:47]: Fixed

File: README.md (hunk 3 above)
Comment (nikever) [reply README.md:57]: OCD-ed!

File: README.md (hunk 4 above)
Comment (nikever) [reply README.md:77]: I feel like you told me this like 10 times. thanks

File: katalog/velero/README.md (hunk 15 above)
Comment (nikever) [reply katalog/velero/README.md]: fixed

File: README.md (hunk 6 above)
Comment (nikever) [reply README.md:103]: fixed

File: README.md (hunk 7 above)
Comment (nikever) [reply README.md:127]: fixed

**Review by ralgozino — APPROVED** [review]
lgtm! 🎉

**Pull request merged**